# config/cepton_test.param.yaml
---

ray_ground_classifier:
  ros__parameters:
    timeout_ms:       10
    cloud_timeout_ms: 110
    raw_topic:       "cepton_cloud"
    ground_topic:    "points_ground"
    nonground_topic: "points_nonground"
    pcl_size:         55000
    frame_id:        "base_link"
    is_structured:    false
    classifier:
      sensor_height_m:                     0.368
      max_local_slope_deg:                 40.0
      max_global_slope_deg:                17.0
      nonground_retro_thresh_deg:          70.0
      min_height_thresh_m:                 0.43
      max_global_height_thresh_m:          3.3
      max_last_local_ground_thresh_m:      3.6
      max_provisional_ground_distance_m:   5.0
      min_height_m:                        -2.5
      max_height_m:                        1.3
    aggregator:
      min_ray_angle_rad: -3.14159
      max_ray_angle_rad:  3.14159
      ray_width_rad:      0.005
      max_ray_points:     512

